Cubical/Data/Fin/Base.agda

```diff
@@ -109,3 +113,29 @@ FinPathℕ : {n : ℕ} (x : Fin n) (y : ℕ) → fst x ≡ y → Σ[ p ∈ _ ] (
 FinPathℕ {n = n} x y p =
     ((fst (snd x)) , (cong (λ y → fst (snd x) + y) (cong suc (sym p)) ∙ snd (snd x)))
   , (Σ≡Prop (λ _ → isProp≤) p)
+
+FinVec : (A : Type ℓ) (n : ℕ) → Type ℓ
+FinVec A n = Fin n → A
```

Comment:
This is not a vector?

Reply:
I just copied the naming in the other definition of Fin (FinData):

https://github.com/agda/cubical/blob/c6a96eeefc6f5f1ce346512d3130de45e4b9a065/Cubical/Data/FinData/Base.agda#L76

Isn't it reasonable-ish?

Reply:
Yes